### PR TITLE
Support get into items and fix hold semantics

### PR DESCRIPTION
**Pull request by pentacular**
Also make unproductive gets fail by default.

### CODE REVIEW

File: api/shape/sort.js [hunk 1]
```diff
@@ -46,10 +42,10 @@ export const sort = Shape.registerMethod3(
       if (limit !== undefined) {
         switch (order) {
           case '>':
-            list = list.filter(({ min }) => min[axis] > limit);
+            leafs = leafs.filter(({ min }) => min[axis] > limit);
             break;
           case '<':
-            list = list.filter(({ max }) => max[axis] < limit);
+            leafs = leafs.filter(({ max }) => max[axis] < limit);
```
Comment (BarbourSmith) [inline api/shape/sort.js:48]: Nice name improvement!

File: api/shape/view.js [hunk 2]
```diff
@@ -53,7 +53,14 @@ export const baseViewOp = async (
   geometry,
   name,
   op = (_x) => (s) => s,
-  { size = 256, inline, width, height, position = [100, -100, 100] } = {}
```
Comment (BarbourSmith) [inline api/shape/view.js:56]: That's cool that there is an option to download views

File: doc/updateNotebook.js [hunk 3]
```diff
@@ -69,47 +69,52 @@ const writeMarkdown = async (
...
+                  `cp '${observedPath}' '${expectedPath}'`
+                );
+              }
+            } catch (error) {
+              console.log(`EE/3: ${JSON.stringify(error)}`);
+              if (error.code === 'ENOENT') {
+                failedExpectations.push(
+                  `cp '${observedPath}' '${expectedPath}'`
+                );
+                failedExpectations.push(`git add '${expectedPath}'`);
```
Comment (BarbourSmith) [inline doc/updateNotebook.js:100]: What is the notebook github integration like?

File: geometry/hold.js [hunk 4]
```diff
@@ -1,6 +1,13 @@
 import { Group } from './Group.js';
-import { inItem } from './inItem.js';
-import { on } from './on.js';
 
-export const hold = (geometry, geometries) =>
-  on(geometry, inItem(geometry), (inside) => Group([inside, ...geometries]));
+export const hold = (geometry, geometries) => {
```
Comment (BarbourSmith) [inline geometry/hold.js:3]: What does hold do?

**Review by BarbourSmith — APPROVED** [review]
Just questions, not really any suggestions or anything, but it looks great to me!

File: doc/updateNotebook.js (hunk 3 above)
Comment (pentacular) [reply doc/updateNotebook.js:100]: Mostly trying to delegate that to the desktop tools.

Adding local directory access next.

File: geometry/hold.js (hunk 4 above)
Comment (pentacular) [reply geometry/hold.js:3]: Ah, it causes an item to include another piece of content.

**Pull request merged**